lib/pubsub/topic.js

```diff
@@ -276,7 +313,10 @@ Topic.prototype.subscribe = function(name, options, callback) {
 
   var path = Subscription.formatName_(this.projectId, name);
   this.makeReq_('PUT', path, null, body, function(err) {
-    if (err) {
+    if (options.reuseExisting && err && err.code === 409) {
```

Comment:
Just add a test for both. In our unit tests, both should be covered. Coverage tool will determine that the branch was taken.